harness/features/allVariables.cloud.test.ts

```diff
@@ -37,6 +38,20 @@ describe('allVariables Tests - Cloud', () => {
                 expect(variablesMap).toMatchObject({})
             })
 
+            it.failing('should throw if variables request fails on user error', async () => {
+                // this will fail until dotnet and node are released
```

Comment:
will this test also pass if this ends up passing once the changes are released?

Reply:
oh right yeah just checked and it fails if it passes - i guess i'll just mark it with skip for now

Reply:
it's better to just merge the failing test in this repo I think. We still haven't figured out the full process of how to write tests over here and then fix them over in the SDK repos, but I think it's better to have failing tests in here that then have to pass on the SDK PR

Reply:
ah true that makes sense